### PR TITLE
Pesolve links of all web/TOC.md and tizen-studio under docs/application

### PR DESCRIPTION
**Pull request by journey2w**
- Fix link errors.
 - Hidden outdated files.

### CODE REVIEW

File: docs/application/native/guides/app-management/efl-ui-app.md [hunk 1]
```diff
@@ -262,4 +262,4 @@ An application with a background running capability must declare the background
 ```
 The `<metadata key="http://tizen.org/metadata/bacgkround-category/<value>"/>` element has no effect on Tizen 2.3 devices, but on Tizen 2.4 and higher devices, it has the same effect as the `<background-category>` element.
 
-The background category of your application can be specified in the [application project settings](../../../../org.tizen.training/html/native/process/setting_properties_n.htm#manifest) in the Tizen Studio.
+The background category of your application can be specified in the [application project settings](../../../../tutorials/process/setting-properties.md#manifest) in the Tizen Studio.
```
Comment (mijin85cho) [inline docs/application/native/guides/app-management/efl-ui-app.md]: ../../../../tutorials/process/setting-properties.md#manifest
=> ../../ looks correct. Also the file name is setting-properties-n.md now.

File: docs/application/tizen-studio/TOC.md [hunk 2]
```diff
@@ -43,45 +44,45 @@
...
 #### [Network Analysis](common-tools/da-network.md)
 #### [OpenGL Analysis](common-tools/da-opengl.md)
-#### [UI Hierarchy Analysis](common-tools/da-ui-hierarchy-n.md)
-### [Call Stack View](native-tools/call-stack-view-n.md)
-### [Static Analyzer](native-tools/static-analyzer-n.md)
-### [Valgrind](native-tools/valgrind-n.md)
-### [T-trace](native-tools/t-trace-n.md)
-### [Address Sanitizer](native-tools/address-sanitizer-n.md)
-### [Code Coverage](native-tools/code-coverage-n.md)
+#### [UI Hierarchy Analysis](common-tools/da-ui-hierarchy.md)
```
Comment (mijin85cho) [inline docs/application/tizen-studio/TOC.md]: Current file name is da-ui-hierarchy-n.md

File: docs/application/tizen-studio/TOC.md [hunk 3]
```diff
@@ -114,14 +115,14 @@
```
Comment (mijin85cho) [inline docs/application/tizen-studio/TOC.md]: Current file name for platform-tools/overview.md is platform-tools/cover-platform.md.

File: docs/application/tizen-studio/extension-sdk/inmovi.md [hunk 4]
```diff
@@ -82,7 +82,7 @@ To use InMobi Tizen Ads in Web applications:
 
 This section describes the InMobi Tizen Ads APIs for Tizen native (EFL) applications. Familiarize yourself with Tizen native application development and EFL UI development before attempting to integrate the Ads SDK.
 
-The InMobi Tizen Ads SDK (inmobi-tizen-adsdk_1.0.3.zip) includes 1 library (Adlib) and 1 header file. To get started, [download the Tizen studio](../setup/download.md). If you are an existing user of Tizen SDK, [migrate from Tizen SDK to Tizen studio](https://developer.tizen.org/development/tizen-studio/download/tizen-studio-migration-guide). For more information on adding extension SDKs, see [Extension SDKs](../setup/advanced-configution.md).
+The InMobi Tizen Ads SDK (inmobi-tizen-adsdk_1.0.3.zip) includes 1 library (Adlib) and 1 header file. To get started, [download the Tizen studio](https://developer.tizen.org/development/tizen-studio/download). For more information on adding extension SDKs, see [Extension SDKs](../setup/advanced-configution.md).
```
Comment (mijin85cho) [inline docs/application/tizen-studio/extension-sdk/inmovi.md]: Current file name for advanced-configution.md is advanced-configuration.md. "ra" in the middle is missing

File: docs/application/tizen-studio/TOC.md [hunk 5]
```diff
@@ -130,7 +131,7 @@
 # [Configurable SDK](configurable-sdk/configurable-sdk.md)
 
 # Release Notes
```
Comment (mijin85cho) [inline docs/application/tizen-studio/TOC.md]: Don't we need to add Release note for 2.2 here? I'm asking as the tizen-studio/index.md page is updated and including  information of 2.2.

File: docs/application/tizen-studio/index.md [hunk 6]
```diff
@@ -1,30 +1,27 @@
...
 
 
 To develop your applications, use the set of tools provided by the Tizen Studio:
 
 - [Setup](setup/installing-sdk.md) Learn about installing and updating the Tizen Studio. You may need to install additional packages to develop applications for various platforms.
 
-- [Native Tools](native-tools/cover-native.md)Learn about the available native application development tools. Native applications are developed using the C programming language. You can create an application project, edit the code to meet your needs, and run the application on the emulator or a target device.
+- [Native Tools](native-tools/overview.md) Learn about the available native application development tools. Native applications are developed using the C programming language. You can create an application project, edit the code to meet your needs, and run the application on the emulator or a target device.
```
Comment (mijin85cho) [inline docs/application/tizen-studio/index.md]: Current file name for overview.md is cover-native.md.

File: docs/application/tizen-studio/index.md [hunk 7]
```diff
@@ -1,30 +1,27 @@
...
 
-- [Web Tools](web-tools/cover-web.md)Learn about the available Web application development tools. Web applications are developed using the JavaScript, HTML, and CSS languages. You can create an application project, edit the code to meet your needs, and run the application on the emulator, simulator, or a target device.
+- [Web Tools](web-tools/overview.md) Learn about the available Web application development tools. Web applications are developed using the JavaScript, HTML, and CSS languages. You can create an application project, edit the code to meet your needs, and run the application on the emulator, simulator, or a target device.
```
Comment (mijin85cho) [inline docs/application/tizen-studio/index.md]: Current file name for overview.md is cover-web.md.

File: docs/application/tizen-studio/index.md [hunk 8]
```diff
@@ -1,30 +1,27 @@
...
 
-- [Platform Tools](platform-tools/cover-platform.md) Learn about the platform development environment that helps you develop preloaded platform applications, platform libraries, and device drivers in Ubuntu.
+- [Platform Tools](platform-tools/overview.md) Learn about the platform development environment that helps you develop preloaded platform applications, platform libraries, and device drivers in Ubuntu.
```
Comment (mijin85cho) [inline docs/application/tizen-studio/index.md]: Current file name for overview.md is cover-platform.md.

File: docs/application/tizen-studio/index.md [hunk 9]
```diff
@@ -81,3 +78,6 @@ The Tizen Studio contains many tools familiar from the old Tizen SDK, but with s
   **Figure: Dynamic Analyzer**
 
   ![Dynamic Analyzer](media/overview_da.png)
+
+
+For more information, see [Release Notes](../release-notes/2_2_release_notes.md).
```
Comment (mijin85cho) [inline docs/application/tizen-studio/index.md]: It looks like ./release-notes/2-2-release-notes.md is correct.

File: docs/application/tizen-studio/web-tools/live-editing.md [hunk 10]
```diff
@@ -24,14 +24,14 @@ To enable live editing in the Tizen Studio:
 If you are running your application for the first time, you must create a new configuration first.
 
 > **Note**  
-> The live editing feature has the following limitations:     
+> The live editing feature has the following limitations:
```
Comment (mijin85cho) [inline docs/application/tizen-studio/web-tools/live-editing.md]: One empty line in here.

File: docs/application/tizen-studio/web-tools/web-simulator.md [hunk 11]
```diff
@@ -116,8 +116,8 @@ In this section, you can modify Google Chrome™ browser-related preferences.
 Enter the full path to the Google Chrome™ browser program. When first started, the Tizen Studio attempts to discover the location. However, it can be necessary to enter or modify this value manually.
 
 - For Linux: `/opt/google/chrome/google-chrome`.
-- For Windows® 64-bit: ` C:\Program Files (x86)\Google\Chrome\Application\chrome.exe`
-- For Windows® 32-bit: ` C:\Program Files\Google\Chrome\Application\chrome.exe`
+- For Windows&regs; 64-bit: ` C:\Program Files (x86)\Google\Chrome\Application\chrome.exe`
```
Comment (mijin85cho) [inline docs/application/tizen-studio/web-tools/web-simulator.md]: `&reg;` is correct. Same issue on line 120.

File: docs/application/native/guides/app-management/efl-ui-app.md (hunk 1 above)
Comment (journey2w) [reply docs/application/native/guides/app-management/efl-ui-app.md]: The name is changed on PR #130.
 
https://github.com/Samsung/tizen-docs/pull/130/commits/d652a344e0b73aec8d7f862e1ff9acaeea1c9985#diff-963d908b28a05830f351dd5bfb091788

File: docs/application/tizen-studio/TOC.md (hunk 2 above)
Comment (journey2w) [reply docs/application/tizen-studio/TOC.md]: I renamed the file name.

File: docs/application/web/guides/TOC.md [hunk 12]
```diff
@@ -1,21 +1,25 @@
...
+### [Overview](app-management/app-info-control.md)
+### [Application Groups](app-management/app-group.md)
+### [Application Preferences](app-management/preference.md)
+### [Application Icons](app-management/app-icons.md)
+### [Application Controls](app-management/appcontrols.md)
+### [Common Application Controls](app-management/common-appcontrols.md)
+### [Widget Information](app-management/widget-info.md)
+### [Package Information](app-management/packages.md)
 ## Application Data Exchange
+### [Overview](app-management/app-communications.md)
```
Comment (mijin85cho) [inline docs/application/web/guides/TOC.md]: Current file name for app-communications.md is app-communication.md. s is missing.

File: docs/application/web/guides/app-management/app-communication.md [hunk 13]
```diff
@@ -4,17 +4,17 @@ The application data exchange features cover the various methods you have to com
 
 You can use the following application data exchange features in your Web applications:
 
-- [Message Port](./message-port.md)
+- [Message Port](message-port.md)
 
   Applications communicate with each other using message port communication. You can send and receive messages using the map data format and trusted or untrusted message port instances.
 
-- [Data Control](./data.md)
+- [Data Control](data.md)
```
Comment (mijin85cho) [inline docs/application/web/guides/app-management/app-communication.md]: Current file name for data.md is data-control.md.

Comment (journey2w) [reply docs/application/web/guides/app-management/app-communication.md]: Done.

File: docs/application/web/guides/TOC.md (hunk 12 above)
Comment (journey2w) [reply docs/application/web/guides/TOC.md]: Done.

**Pull request merged**